diff --git a/binding.gyp b/binding.gyp
index 0ec694d..67f6e8a 100644
--- a/binding.gyp
+++ b/binding.gyp
@@ -7,7 +7,7 @@
         "<!@(node -p \"require('node-addon-api').include\")"
       ],
       'cflags!': [ '-fno-exceptions' ],
-      'cflags_cc!': [ '-fno-exceptions' ],
+      'cflags_cc!': [ '-fno-exceptions', '-std=gnu++20' ],
       'xcode_settings': {
         'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
         'CLANG_CXX_LIBRARY': 'libc++',
@@ -35,6 +35,7 @@
         }],
         ['OS=="linux"', {
           "sources": ["src/FontManagerLinux.cc"],
+          'cflags_cc': ['-std=gnu++17'],
           "link_settings": {
             "libraries": ["-lfontconfig"]
           }
